src/schemas/link.ts

```diff
@@ -0,0 +1,13 @@
+import { model, Schema, Types } from "mongoose";
+
+export interface Link {
+    _id: string,
+    name: string,
+    url: string,
+}
+
+const linkSchema = new Schema({
+    name: { type: String, required: true },
+    url: { type: String, required: true },
+});
+export const linkModel = model<Link>('Link', linkSchema);
```

Comment:
We did technically say you didn't have to use an ORM, but I'm good with Mongoose. I've written pure MDB Atlas interaction logic and it was a nightmare to maintain.